packages/repeater/src/bus/EventBusFactory.spec.ts

```diff
@@ -0,0 +1,44 @@
+import 'reflect-metadata';
+import { EventBusFactory } from './EventBusFactory';
+import { Configuration, Credentials } from '@secbox/core';
+import { instance, mock, reset, when } from 'ts-mockito';
+import { RMQEventBus } from '@secbox/bus';
+import { container } from 'tsyringe';
+
+describe('EventBusFactory', () => {
+  const token = 'dummmmy.nexa.vennegtzr2h7urpxgtksetz2kwppdgj0';
+  const MockedConfiguration = mock<Configuration>(Configuration);
```

Comment:
Use camel case instead, since a result of execution of mock is not constructor
```suggestion
  const mockedConfiguration = mock(Configuration);
```